apps/sasquatch/src/projectDependency/java/com/microsoft/appcenter/sasquatch/activities/StorageActivity.java

```diff
@@ -94,33 +97,22 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
     private void updateStorageType(int position) {
         switch (position) {
             case 0:
-                listView.setAdapter(new ArrayAdapter<String>(this, android.R.layout.simple_list_item_1, appDocumentList));
+                listView.setAdapter(new ArrayAdapter<String>(this, R.layout.item_view_app, appDocumentList));
                 break;
             case 1:
                 if (AuthenticationProviderActivity.userID != null) {
-                    listView.setAdapter(new ArrayAdapter<String>(this, android.R.layout.simple_list_item_1, userDocumentList));
+                    CustomItemAdapter adapterUser = new CustomItemAdapter(userDocumentList, this);
+                    listView.setAdapter(adapterUser);
                 } else {
-                    String[] signInReminder = {"Please Sign In Identity First Before Get User Document!"};
-                    listView.setAdapter(new ArrayAdapter<String>(this, android.R.layout.simple_list_item_1, signInReminder));
+                    ArrayList<String> signInReminder = new ArrayList<String>() {{
+                        add("Please Sign In Identity First Before Get User Document!");
```

Comment:
Never hardcode text displayed in UI, use strings.xml

Reply:
Fix in new commit.